.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Feature request
+about: Propose a feature you would like to see added in the software
+
+---
+
+<!--
+Thank you for your feature request.
+
+NOTE:
+
+    If you're asking about how to use OpenSSL, this isn't the right 
+    forum.  Please see our
+    [User Support resources](https://github.com/openssl/openssl/blob/master/.github/SUPPORT.md)
+
```

Comment:
Side note: I think it would beneficial to ask the users to (optionally, if relevant) include (a list of) the OpenSSL branch(es) for which the feature request is meant (it would ease tagging and reviewing).

Reply:
E.g. It's not that unusual to open feature requests that are strictly 1.0.2 (e.g. #7622) , or 1.1.1 rather than for `master`.
Something vague like "next-major-release" might also be a potential valid value for this field.

Reply:
I'm thinking that for the new user, asking for branches is more confusion than useful.  The version they did experience an issue with is already good enough, and we can build the case for other versions if necessary through conversation.